src/scenes/gamebanana.lua

```diff
@@ -283,7 +283,7 @@ function scene.loadPage(page)
                 cacheable = false
             }):with(uiu.bottombound(16)):with(uiu.rightbound(16)):as("error"))
             scene.loadingPage = nil
-            pagePrev.enabled = not isQuery and page > 0 and ((scene.sort == "latest" and scene.itemtypeFilter.filtervalue == "") or page > 1)
+            pagePrev.enabled = not isQuery and page > 0 and ((scene.sort == "latest" and next(scene.itemtypeFilter) == nil) or page > 1)
```

Comment:
Bit confused what the `next` call here is trying to achieve 😓 Is this just for getting the first index of `scene.itemtypeFilter`? In that case, accessing a table at a key/index that doesn't exist does not error and gives nil.